### ISSUE TITLE
Running locally error: No body was attached to the request

### ISSUE THREAD

**Issue by Pancat007** [Azure-Samples/azure-search-openai-demo#790]
> Please provide us with the following information:
> ---------------------------------------------------------------

### This issue is for a: (mark with an `x`)
```
- [ ] bug report -> please search issues before submitting
- [ ] feature request
- [ ] documentation issue or request
- [ ] regression (a behavior that used to work and stopped in a new release)
```

### Minimal steps to reproduce
> 
run ./start.ps1

### Any log messages given by the failure
>
Serving Quart app 'app'
 * Debug mode: False
 * Please use an ASGI server (e.g. Hypercorn) directly in production
 * Running on http://localhost:50505 (CTRL + C to quit)
WARNING:azure.identity.aio._credentials.environment:Incomplete environment configuration. These variables are set: AZURE_TENANT_ID
INFO:azure.identity.aio._credentials.managed_identity:ManagedIdentityCredential will use IMDS
INFO:azure.core.pipeline.policies.http_logging_policy:Request URL: 'http://x.x.x.x/metadata/identity/oauth2/token?api-version=REDACTED&resource=REDACTED'
Request method: 'GET'
Request headers:
    'User-Agent': 'azsdk-python-identity/1.14.1 Python/3.10.11 (Windows-10-10.0.22621-SP0)'
No body was attached to the request

### Expected/desired behavior
>

### OS and Version?
> Windows 7, 8 or 10. Linux (which distribution). macOS (Yosemite? El Capitan? Sierra?)
win 11

### azd version?
> run `azd version` and copy paste here.
azd version 1.3.1 (commit b5030da0f29ffe98664c40450187fd8a4cb0d157)

### Versions
>

### Mention any other details that might be useful

> ---------------------------------------------------------------
> Thanks! We'll be in touch soon.

**Comment by pamelafox** [comment]
I don't think that "No body was attached to the request" is necessarily an error, I see that as well for some calls. Does something else indicate there's an error?

**Comment by adamcodes716** [comment]
@pamelafox I am getting the exact same error.  The problem is that nothing loads in the browser:

```
Request headers:
    'User-Agent': 'azsdk-python-identity/1.14.0 Python/3.11.6 (Windows-10-10.0.22000-SP0)'
No body was attached to the request
INFO:azure.identity.aio._credentials.chained:DefaultAzureCredential acquired a token from AzureCliCredential
[2023-11-02 11:35:26 -0400] [33696] [INFO] Running on http://127.0.0.1:50505 (CTRL + C to quit)
INFO:hypercorn.error:Running on http://127.0.0.1:50505 (CTRL + C to quit)

![image](https://github.com/Azure-Samples/azure-search-openai-demo/assets/63405553/62b5aa20-96b2-40c1-8563-f9f0e6ef2c9c)


```

**Comment by pamelafox** [comment]
Please try the mimetypes code suggested here and see if that helps:
https://stackoverflow.com/a/70741738/1347623

**Comment by hsamha** [comment]
Forcing the Python code to use IPv4 resolved the issue for me:

```
import socket
import requests.packages.urllib3.util.connection as urllib3_cn

def allowed_gai_family():
    return socket.AF_INET  # Force IPv4

urllib3_cn.allowed_gai_family = allowed_gai_family
```

**Comment by pamelafox** [comment]
We fixed the original issue (it was mime-types related), closing this.
